README.md

```diff
@@ -30,112 +30,108 @@ To facilitate robust API governance, we're excited to introduce **API Analysis**
+The Azure Developer CLI (`azd`) is a developer-centric command-line interface (CLI) tool for creating Azure applications.
 
-    ```bash
-    az login
-    azd auth login
-    ```
+You need to install it before running and deploying with the Azure Developer CLI.
 
-1. Initialize your Azure environment by running the command below:
+### Windows
 
-    ```bash
-    azd init
-    ```
+```powershell
+powershell -ex AllSigned -c "Invoke-RestMethod 'https://aka.ms/install-azd.ps1' | Invoke-Expression"
+```
 
-   While you're running `azd init`, you will be asked to provide:
+### Linux/MacOS
 
-   - How to initialize: Choose the `Select a template` option, then `Y`.
-   - A template: Choose the `Minimal template` option. <!-- This option will be changed to "API Center Analyzer" once this PR is merged, which is not available at this time of PR -->
-   - An environment name: Any name you want to give to your environment.
+```
+curl -fsSL https://aka.ms/install-azd.sh | bash
+```
 
-1. Run the following command to provision and deploy this analyzer to Azure:
+After logging in with the following command, you will be able to use the `azd` cli to quickly provision and deploy the application.
 
-    ```bash
-    azd up
-    ```
+```
+# authenticate to Azure Developer CLI
+azd auth login
 
-   While you're running `azd up`, you will be asked to provide:
+# authenticate to Azure CLI
+az login
+```
 
-   - Azure subscription
-   - Location to provision resources
-   - The `apicName` parameter: Enter the API Center instance name to connect the analyzer to &ndash; if it's provided, the analyzer will be attached to the API Center instance; otherwise, a new API Center instance will be provisioned and connected.
-   - The `useMonitoring` parameter: Choose `yes` or `no` &ndash; if you choose `yes`, both Application Insights and Log Analytics will be provisioned; otherwise, both won't be provisioned.
+Then, execute the `azd init` command to initialize the environment (You do not need to run this command if you already have the code or have opened this in a Codespace or DevContainer).
+```
+azd init -t https://github.com/Azure/APICenter-Analyzer
+```
+Enter an environment name.
 
-   You will get the following Azure resources provisioned:
+(Optional) If you want to use an existing api center resource instead of creating a new one, you can run the following command to set its name and its resource group name as environment variables.
+```
+azd env set APIC_NAME <your-api-center-name>
 
-   - Azure Functions
-   - Storage Account
-   - API Center (if `apicName` is not provided)
-   - Application Insights (if `useMonitoring` is set to `yes`)
-   - Log Analytics (if `useMonitoring` is set to `yes`)
+azd env set APIC_RESOURCE_GROUP_NAME <your-api-center-resource-group-name>
+```
+>You can set the environment variable `USE_MONITORING` to `yes` or `no` to decide whether create monitoring resources or not. The default value is `yes`.
```

Comment:
It seems there is no way to directly allow the user to pass an empty parameter when `azd up`.
If we create the parameters without giving them a blank value like this: `param apiCenterName string`, it will force users to pass a value that can't be blank.
In this case, we can only select existing resources and are unable to create new resources.

Reply:
Well, if we don't use `azd env set` and set the parameter, and put the parameter without the default empty value like `param apiCenterName string`, it will ask devs to enter the name. If devs enter nothing, it will be passed the empty string. That's what my bicep does.

Reply:
Sorry for not noticing that. I will remove the default values.